### PR TITLE
Mag 34 remove location

### PR DESCRIPTION
**Pull request by gminaya**
Remove location from an existing campaign functionality added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env.old
```
Comment (amhed) [inline .gitignore:16]: Noooooooooooooooo 💀

File: src/components/CampaignsAdmin/edit/CampaignDetails.tsx [hunk 2]
```diff
@@ -87,6 +82,18 @@ export function CampaignDetails() {
     }
     return data;
   };
+  
+  /**
+     * Remove location from campaign
+     */
+  const removeAddedLocation = (locationId:number, locationCONF: CampaignLocationInfo[] | undefined) => {
```
Comment (amhed) [inline src/components/CampaignsAdmin/edit/CampaignDetails.tsx:89]: Why was this needed?

Comment (amhed) [reply src/components/CampaignsAdmin/edit/CampaignDetails.tsx:89]: ohhhh now I understand. I think its best to not physically remove rows from the database. We can mark them deleted instead and keep the record

Comment (gminaya) [reply src/components/CampaignsAdmin/edit/CampaignDetails.tsx:89]: That functionality does not delete a location from the DB. It removes a location from the locations list assigned to a campaign. This is important because the user may add a location to a campaign by mistake.
